app/Http/Controllers/Wiki/LanguagesController.php

```diff
@@ -153,9 +153,7 @@ function () use (
                         'languages.rolv_code',
                         \DB::raw($select_country_population . ' as country_population')
                     ])
-                    ->with(['bibles' => function ($query) {
-                        $query->whereHas('filesets');
```

Comment:
We don't need to validate if bibles have filesets attached because we are doing it into the isContentAvailable